### PR TITLE
#16 admin panel

### LINKED ISSUE #16: Panel Administratora
- [ ] Użytkownik z konta administratora ma możliwość zaakceptowania nowych restauracji.
- [ ] Użytkownik z konta administratora ma możliwość usuwania pojedynczych komentarzy dla danej lodziarni.

### PR DESCRIPTION
**Pull request by Ola-M**
Closes #16

### CODE REVIEW

**Comment by Ola-M** [comment]
image
image
image

File: freezy-ice-web/src/Components/AdminPanel/AcceptingShopDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+            display: 'flex',
+            justifyContent: 'right',
+            textAlign: 'right',
+        },
+        rate: {
+            paddingTop: '10%',
+        },
+        hours: {
+            margin: '5px',
+        },
```
Comment (Skrabka98) [inline freezy-ice-web/src/Components/AdminPanel/AcceptingShopDetails.tsx]: Unused

File: freezy-ice-web/src/Components/Modals/CategoryModal.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        rateFrame: {
+            display: 'flex',
+            justifyContent: 'right',
+            textAlign: 'right',
+        },
+        rate: {
+            paddingTop: '10%',
+        },
+    }),
+);
```
Comment (Skrabka98) [inline freezy-ice-web/src/Components/Modals/CategoryModal.tsx]: unused

File: freezy-ice-web/src/Store/Reducer/ShopAdmin/action.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    });
+    await GetNotApprovedShops(currentPage);
+}
+
+export async function GetUnacceptedShops(dispatch: any, currentPage: number) {
+    dispatch({
+        type: ActionsEnums.LOADING,
+    });
+    const result = await GetNotApprovedShops(currentPage);
+    console.log('action', result);
```
Comment (Skrabka98) [inline freezy-ice-web/src/Store/Reducer/ShopAdmin/action.ts]: log

**Review by Skrabka98 — CHANGES_REQUESTED** [review]

**Review by Skrabka98 — APPROVED** [review]

**Pull request merged**